custom/icds_reports/models/aggregate.py

```diff
@@ -347,6 +349,51 @@ class ChildHealthMonthly(models.Model, AggregateMixin):
     lunch_count = models.IntegerField(blank=True, null=True)
     state_id = models.TextField(blank=True, null=True)
     opened_on = models.DateField(blank=True, null=True)
+    birth_weight = models.PositiveSmallIntegerField(null=True, help_text="birth weight in grams")
+    mother_id = models.TextField(blank=True, null=True)
+    due_list_date_1g_dpt_1 = models.DateField(blank=True, null=True)
```

Comment:
Oh wow! there is a lot of these. Yeah, I think having a separate vaccine table might be beneficial. If necessary we can even include the columns in the `child_health_monthly_view` so that from the front end you can't tell the difference but this a ton of data to dump into this one table